user-guide/scheduler.rst

```diff
@@ -600,24 +609,33 @@ specify other types of MPI task placement. For example, setting it to
 
 ::
 
-    salloc --nodes=8 --tasks-per-node=2 --cpus-per-task=1 --time=0:10:0 --account=t01
-
-    salloc: Granted job allocation 24236
-    salloc: Waiting for resource configuration
-    salloc: Nodes cn13 are ready for job
-
-    module load xthi
-    export MPICH_RANK_REORDER_METHOD=0
-    srun --cpu-bind=core xthi
-
-    Hello from rank 0, on nid01041. (core affinity = 0-63)
-    Hello from rank 1, on nid01111. (core affinity = 0-63)
-    Hello from rank 2, on nid01118. (core affinity = 0-63)
-    Hello from rank 3, on nid01282. (core affinity = 0-63)
-    Hello from rank 4, on nid01041. (core affinity = 0-63)
-    Hello from rank 5, on nid01111. (core affinity = 0-63)
-    Hello from rank 6, on nid01118. (core affinity = 0-63)
-    Hello from rank 7, on nid01282. (core affinity = 0-63)
+  salloc --nodes=8 --tasks-per-node=2 --cpus-per-task=1 --time=0:10:0 --account=t01
+
+  salloc: Granted job allocation 24236
+  salloc: Waiting for resource configuration
+  salloc: Nodes cn13 are ready for job
+
+  module load xthi
+  export OMP_NUM_THREADS=1
+  export MPICH_RANK_REORDER_METHOD=0
+  srun --cpu-bind=core xthi
```

Comment:
It should be _--cpu-bind=cores_

Reply:
Fixed in https://github.com/ARCHER2-HPC/archer2-docs/pull/48/commits/e5b33cb6bab8e9f6412ff58249049a5d0cab48bf